src/sagemaker_mxnet_container/default_save.py

```diff
@@ -37,10 +38,11 @@ def save(model_dir, model):
         model_dir (str): the directory for saving the model
         model (mxnet.mod.Module): the module to be saved
     """
-    model.symbol.save(os.path.join(model_dir, SYMBOL_PATH))
-    model.save_params(os.path.join(model_dir, PARAMS_PATH))
-
-    signature = [{'name': data_desc.name, 'shape': [dim for dim in data_desc.shape]}
-                 for data_desc in model.data_shapes]
-    with open(os.path.join(model_dir, SHAPES_PATH), 'w') as f:
-        json.dump(signature, f)
+    if current_host == hosts[0]:
```

Comment:
Let's add a comment here about the first host is the master.

Reply:
given that I'm moving `scheduler_host()` to this module, using that method will help document the situation